mzroll/projectdockwidget.cpp

```diff
@@ -48,6 +49,12 @@ ProjectDockWidget::ProjectDockWidget(QMainWindow *parent):
     //Trigger to open() in slot to load samples while uploading .mzroll file --@Giridhari
     connect(loadButton,SIGNAL(clicked()),_mainwindow, SLOT(open()));
 
+    QToolButton* loadSetsButton = new QToolButton(toolBar);
+    loadSetsButton->setIcon(QIcon(rsrcPath + "/fileopen.png"));

```

Comment:
@shubhra-agrawal Icon is same as loading samples/project file. Please use some different icon.